pkg/api/postage.go

```diff
@@ -276,7 +276,7 @@ func (s *Service) postageGetStampBucketsHandler(w http.ResponseWriter, r *http.R
 	}
 	hexBatchID := hex.EncodeToString(paths.BatchID)
 
-	issuer, save, err := s.post.GetStampIssuer(paths.BatchID)
+	issuer, _, err := s.post.GetStampIssuer(paths.BatchID)
```

Comment:
Not sure about this. Why is the `save` func ignored?

Reply:
because the stamp issuer is not updated, it's a read only operation